src/components/media-player/media-player-track-group.js

```diff
@@ -230,9 +240,16 @@ function MediaPlayerTrackGroup({
     }));
   };
 
-  return (
-    <Fragment>
-      <MediaPlayerTrack
+  const getTrackComponent = sourceUrl => {
+    const isAudioFile = getResourceType(sourceUrl) === RESOURCE_TYPE.audio;
+    const isInternalFile = getMediaSourceType({ sourceUrl, cdnRootUrl }) === MEDIA_SOURCE_TYPE.internal;
+    return isMultitrack && isInternalFile && isAudioFile ? PreloadingMediaPlayerTrack : MediaPlayerTrack;
```

Comment:
So using the `PreloadingMediaPlayerTrack` would not work for video files in the end then, right?

Reply:
It would, but we might end up preloading huge files into memory (with audio it's usually just MP3 with a few megabytes).

Reply:
Indeed. Though big files also carry the bigger risk for the tracks to get out of sync. But let's see what usage feedback comes in.